azure-pipelines.yml

```diff
@@ -69,7 +72,11 @@ jobs:
     steps:
       - script: |
           curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly
-        displayName: 'Install rust'
+          rustc --version
+          rustup toolchain install nightly
+          rustup default nightly
+          rustc --version
+        displayName: 'Install rust, select nightly'
```

Comment:
@mikelodder7 this is how I worked around https://github.com/rust-lang/rustup/issues/1079